admin/resources/js/components/form/Input.tsx

```diff
@@ -1,11 +1,14 @@
 import * as React from "react";
 import get from "lodash/get";
 import { RegisterOptions, useFormContext } from "react-hook-form";
+import InputWrapper from "../H2Components/InputWrapper";
 
 export interface InputProps
```

Comment:
Should the type be added to the interface for InputProps and limited to only a few, say the ones you've added as storybook stories (text, email, tel) of the [types available](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types)?